packages/p2p-media-loader-hlsjs/src/services/fragment-loader.ts

```diff
@@ -54,33 +67,29 @@ export class FragmentLoaderBase implements Loader<FragmentLoaderContext> {
     }
     try {
       this.response = await this.fetchSegment(context.url, byteRange);
+      loading.first = performance.now();
     } catch (error) {
       if (!this.stats.aborted) {
         return this.handleError(error as { code: number; text: string });
       }
     }
     if (!this.response) return;
-    const { loading } = stats;
-    const loadedBytes = this.response.data.byteLength;
-    loading.first = performance.now();
-    loading.end = performance.now() + 1;
-
-    const { bandwidth, loadingStartTime } = this.getLoadingStatByTargetBitrate({
-      targetLevelBitrate: 1650064,
-      aboveLevelBitrate: 2749539,
-      loadingEndTime: loading.first,
+    const data = await this.response.arrayBuffer();
+    loading.end = performance.now();
+    const loadedBytes = data.byteLength;
+
+    loading.start = this.getLoadingStatByTargetBitrate({
```

Comment:
I think that `start` affects only latency. Bitrate is calculated between `end` and `first`.